browser/src/control/Control.Notebookbar.js

```diff
@@ -375,6 +375,17 @@ L.Control.Notebookbar = L.Control.extend({
 		}
 	},
 
+	ignoreContextChange(contextes) {
```

Comment:
please consider renaming to:
shouldIgnoreContextChange - as it returns boolean
ignoreContextChange - sounds like void, "do it"